GVFS/GVFS.Platform.Windows/ProjFSFilter.cs

```diff
@@ -373,7 +370,7 @@ public bool IsReady(JsonTracer tracer, string enlistmentRoot, out string error)
             return
                 IsServiceRunning(tracer) &&
                 IsNativeLibInstalled(tracer, new PhysicalFileSystem()) &&
-                TryAttach(tracer, enlistmentRoot, out error);
+                TryAttach(enlistmentRoot, out error);

```

Comment:
It looks like the error being returned is not being logged by the MountVerb:

```
if (!GVFSPlatform.Instance.KernelDriver.IsReady(tracer, enlistment.EnlistmentRoot, out errorMessage))
{
    tracer.RelatedInfo($"{nameof(MountVerb)}.{nameof(this.Execute)}: Enabling and attaching ProjFS through service");

    if (!this.ShowStatusWhileRunning(
        () => { return this.TryEnableAndAttachPrjFltThroughService(enlistment.EnlistmentRoot, out errorMessage); },
        $"Attaching ProjFS to volume"))
    {
```

Could you add this error message here:

```
    tracer.RelatedInfo($"{nameof(MountVerb)}.{nameof(this.Execute)}: Enabling and attaching ProjFS through service");
```

So that we can see in the mount verb logs why GVFS decided to ask the service?

Reply:
@wilbaker Let me know what you think of the log message.  That sort of thing is not my forte :-).

Reply:
@wilbaker Let me know what you think of the log message. That sort of thing is not my forte :-).

Reply:
I think the message sounds good, but I think we can combine these into a single call and use `EventMetadata`.  I'll add a comment by the new code with what I mean.